.github/workflows/build-android.yml

```diff
@@ -46,16 +49,16 @@ jobs:
       - name: Install node_modules for example/
         run: yarn install --frozen-lockfile --cwd example
 
-      - name: Restore Gradle cache
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/.gradle/caches
-            ~/.gradle/wrapper
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
-          restore-keys: |
-            ${{ runner.os }}-gradle-
-      - name: Run Gradle Build for android/
-        run: cd android && ./gradlew assembleDebug --build-cache && cd ..
+      # - name: Restore Gradle cache
```

Comment:
Why it's commented out?

Reply:
setup-java@v3 already has a cache logic integrated, see line 33